### DISCUSSION TITLE
Raspian, iso or hints?

### DISCUSSION THREAD

**Discussion by aerodan** [trunkmaster/nextspace#417] [General]
Anyone out there successfully installed on Bullseye 64bit Raspian (Pi 4)? The current build fails libobj2c as well as other libraries in the Packages/Source and Packages/Debian directories. I've worked through a full compile of Gnustep, and lib replacement using what's in that distro, but getting Nextspace to compile is a no go. Trunkmaster, amazing work on this project, you are a rare talent and I wish you safety and freedom.

**Comment by felixding** [comment]
Take a look at this: https://ivmoreau.com/almost-ubuntu-nextstep/

After installing, extract [Library.tar.gz](https://github.com/trunkmaster/nextspace/files/6017866/Library.tar.gz) to your home directory, reboot to command line, run `sudo systemctl start loginwindow` and you'll get to the desktop.

NEXTSPACE is still extremely buggy though.

**Reply by aerodan** [reply]
Thanks for this pointer, very helpful. I'm shelving any debian installs, but maybe Fedora x86 for fun later.
